frame/tips/src/benchmarking.rs

```diff
@@ -112,8 +109,8 @@ benchmarks! {
 	}: _(RawOrigin::Signed(caller), hash)
 
 	tip_new {
-		let r in 0 .. MAX_BYTES;
-		let t in 1 .. MAX_TIPPERS;
+		let r in 0 .. T::MaximumReasonLength::get();
+		let t in 1 .. T::Tippers::max_len() as u32;
```

Comment:
maybe it also worth mentionning it that ContainsLengthBound::max_len implementation of Tippers is used in benchmark.